apps/api-frontend/src/app/pages/search-pages/search.css

```diff
@@ -1,3 +1,14 @@
 .app .search p {
   text-align: center;
-}
\ No newline at end of file
+}
+
+.app .search .search-options {
+  display: grid;
+  width: 100%;
+  align-items: center;
+  justify-content: left;
+  padding: 0.5rem 1rem;
+  border: 1px dotted black;
+  margin: 1rem 0;
+  border-radius: 0.5rem;
```

Comment:
nit: styleguide prefers to alphabetize the properties - I'll leave that up to you, but do highly recommend it (it comes in especially handy as apps grow)

Reply:
Done (ran through the alphabet a few times in my head :) )